applications/FluidDynamicsApplication/trilinos_extension/custom_python/fluid_dynamics_trilinos_extension.cpp

```diff
@@ -0,0 +1,35 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//
+
+#ifdef KRATOS_PYTHON
+#include <pybind11/pybind11.h>
+
+#include "add_trilinos_processes_to_python.h"
+#include "add_trilinos_schemes_to_python.h"
+#include "add_trilinos_strategies_to_python.h"
+#include "add_trilinos_utilities_to_python.h"
```

Comment:
not giving the full path should not be a problem right?

Reply:
I don't think so, because they are on the same folder as the cpp in question, but I can add the path if preferred.